scripts/test_shared_ci.py

```
@@ -0,0 +1,145 @@
+
+    module.joinpath("go.mod").unlink()
+    for name in ("one", "two"):
+        child = module / name
+        child.mkdir()
+        child.joinpath("go.mod").write_text(f"module example.com/{name}\\n", encoding="utf-8")
+    module.joinpath("go.work").write_text("go 1.26\\nuse ./one\\nuse ./two\\n", encoding="utf-8")
+    log.write_text("", encoding="utf-8")
+    rendered = script.replace("${{inputs.go-version}}", "1.27.0")
+    for source, target in replacements.items():
+        rendered = rendered.replace(source, target)
+    subprocess.run(["sh", "-eu", "-c", rendered], check=True, env=environment)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)